src/routes/authRoutes.js

```diff
@@ -1,9 +1,10 @@
 const express = require('express');
 const router = express.Router();
 
-const { register, login } = require('../controllers/authController');
+const { register, login, logout } = require('../controllers/authController');
 
 router.post('/register', register);
 router.post('/login', login);
+router.get('/logout', logout);
```

Comment:
Per our convo in slack:

As for using `POST` vs `GET` for logout. The latest recommendation is to use `POST` because POST means “changing state in the backend”, vs GET is used to “retrieve state in backend”.. We are changing state because we are updating the cookies and changing the user from being logged into not being logged in. Even though we’re not sending any request body, POST is also better because sometimes browsers will try to eager load `GET` requests or cache them; and we don’t want the browser doing that for the logout functionality. We only want to logout when the user has explicitly requested that

Reply:
I have changed the logout router for POST commit d57c739ed60c5d19b29d92430eeab9e723381c8a